client/src/App.js

```diff
@@ -1,12 +1,41 @@
 import React from 'react';
 import './App.css';
+import Home from './Home';
+import Header from './Header';
+import About from './About'
+import Curious from './Curious'
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link,
+  NavLink,
+  HashRouter
+} from "react-router-dom";
+
+
 
 function App() {
   return (
-    <div className="App">
-      <h1>Hello portfolio!</h1>
-    </div>
+    <HashRouter>
+      <div>
+      <Header />
+      <ul className='header'>
+        <li><NavLink exact to={'/'} component={Home} /></li>
+        <li><NavLink exact to={'/about'} component={About} /></li>
+        <li><NavLink exact to={'/curious'} component={Curious} /></li>
+      </ul>
+      <div className='content'>
+        <Route exact path='/' component={Home} />
+        <Route path='/about' component={About} />
+        <Route path='/curious' component={Curious} />
+      </div>
+      </div>
+    </HashRouter>
```

Comment:
It looks like you are trying to create your links and your react router here. In the base repo from Bob [here](https://github.com/BurlingtonCodeAcademy/personal-portfolio-Habenzy/), there is an example of how to set up a react router in [App.js](https://github.com/BurlingtonCodeAcademy/personal-portfolio-Habenzy/blob/master/client/src/App.js) and he sets up his links in his [Header.js](https://github.com/BurlingtonCodeAcademy/personal-portfolio-Habenzy/blob/master/client/src/Header.js). You should be able to set up a basic router with links from that without using the HashRouter or NavLink components.